packages/common/src/types/validation.ts

```diff
@@ -1,8 +1,11 @@
+import type { ChartKind } from './savedCharts';
```

Comment:
this resolves a circular dependency between this file and `packages/common/src/types/savedCharts.ts` (which imports `ValidationSummary` from `validation.ts`) so we just import the type definition. 
What do you think?
If not, what should I do here? I still want to reference `ChartKind` enum here...

Reply:
I think it's fine like this
@ZeRego  ??